src/Raven.Server/Web/System/CompareExchangeHandler.cs

```diff
@@ -34,6 +34,29 @@ public Task GetCompareExchangeValues()
             return Task.CompletedTask;
         }
 
+        [RavenAction("/databases/*/cmpxchg/indexes", "GET", AuthorizationStatus.ValidUser)]
+        public Task GetCompareExchangeIndexes()
+        {
+            var keys = GetStringValuesQueryString("keys");
```

Comment:
Use the query string ability to send mulitple values to avoid doing string splits